### ISSUE TITLE
Freee crashes when loading savegame

### ISSUE THREAD

**Issue by Skyburn1** [ekolis/FrEee#52]
Freee crashes when I try to load turn 24 in the Freee MegaBrawl game.

![freeeerror2](https://user-images.githubusercontent.com/59808234/72684380-3a984680-3aae-11ea-9dc4-2639a7c09403.png)
![freeeerror](https://user-images.githubusercontent.com/59808234/72684381-3a984680-3aae-11ea-95a0-968b0d3d25dc.png)
[errorlog.txt](https://github.com/ekolis/FrEee/files/4082908/errorlog.txt)

**Comment by ekolis** [comment]
Hmm. CW reported a similar error, but I can load the save just fine. Have you tried downloading the latest version of the game?

**Comment by Skyburn1** [comment]
I did a pull right before loading

**Comment by ekolis** [comment]
Pull and build?

**Comment by Skyburn1** [comment]
yes, I hit start and it builds Freee and then runs

**Comment by ekolis** [comment]
Just on a lark... Can you zip up the GAM file you are trying to open? Perhaps something got corrupted in transit and somehow you have a different file from the one I'm trying to open...

**Comment by Skyburn1** [comment]
Ok.

[fmbrawl_24_0002.zip](https://github.com/ekolis/FrEee/files/4083516/fmbrawl_24_0002.zip)

**Comment by ekolis** [comment]
Nope, game doesn't crash for me. I made sure I don't have any changes that I forgot to push. You are pulling from github and not bitbucket, right?

**Comment by Skyburn1** [comment]
Hmm, let me check

**Comment by Skyburn1** [comment]
Yeah, it's bitbucket. What's the right repository for github?

**Comment by Skyburn1** [comment]
Actually, it's saying unspecified, but the last thing I have is your note from 11 days ago that you're moving the project to github

**Comment by ekolis** [comment]
This repo here is the correct repo; you can clone from this git url: https://github.com/ekolis/FrEee.git - you can do this directly within Visual Studio either in Team Explorer or (in VS2019) the start screen.

**Comment by ekolis** [comment]
Looks like the problem was that Skyburn was trying to access the old bitbucket repo which does not have the latest code.
